src/app/(mobile-ui)/home/page.tsx

```diff
@@ -29,6 +39,16 @@ export default function Home() {
 
     const totalCards = hasWallets ? wallets.length + 1 : 1
 
+    // hide balance
+    const handleToggleBalanceVisibility = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.stopPropagation()
```

Comment:
Q: What is e.stopPropagation for here?

Reply:
@Hugo0 added this to prevent bubbling, if you clicked on the hide button, it would take you to the respective wallet page, which shouldn't happen as all I wanted was to hide the balance, so to prevent that from happening, I added this `stopPropagation` method